### DISCUSSION TITLE
Using Logitech G920 without an editor

### DISCUSSION THREAD

**Discussion by tashahughes** [microsoft/AirSim#3412] [Support Q&A] [answered]
I am very new to AirSim/Unreal, so I apologise for my lack of knowledge, but I just wanted to ask if there was a way of using the steering wheel without having the editor present. I managed to connect the steering wheel to the Landscape Mountains environment successfully, but I wanted to use the CityEnviron environment. The files for 'CityEnviron' are missing the source code, (I assume because its not a full version), but I can still run the environment and get the data I need using the keyboard.

 I'd much prefer to use the steering wheel over keyboard, but I don't know how I can get Unreal to pick up the steering wheel without having an editor? Is there any way to get the editor up from the other environments provided or is there a way to get this environment to detect the steering wheel?

**Comment by zimmy87** [comment]
Hi @tashahughes, can you repost your solution as a comment and mark it as the answer so this doesn't show as unanswered anymore?

**Comment by tashahughes** [comment, marked as answer]
I solved this problem but just in case anybody has the same issue in the future, you have to edit the settings.json file. Follow the docs in that file but change the PhysX file so that instead of -1 it is 1, (providing you have connected your controller already) This seems to pick up any connected controller that is not the keyboard
